lib/xayn_card_view/card_view.dart

```diff
@@ -85,6 +85,8 @@ abstract class CardViewAnimatedState extends AnimatedWidgetBaseState<CardView> {
   Tween<EdgeInsets>? _padding;
   Tween<BorderRadius>? _clipBorderRadius;
 
+  BoxConstraints? get lastKnownConstraints;
```

Comment:
it looks like it could be `@protected`

Reply:
Also, it might be good to add a comment on why we need it. From my understanding, we only need it to get the `targetPosition` in case of triggering `jump`.

Reply:
yes, good point